### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by everyonesdesign**
- Add multiple events binding (cool Backbone style notation)
- Update gitignore, so now builds don't go to git
- Add script for browserify building via `npm run build`

### CODE REVIEW

**Comment by everyonesdesign** [comment]
@HenrikJoreteg
by the way, why there's no issues in this repo?

File: README.md [hunk 1]
```diff
@@ -96,6 +96,16 @@ events.bind('click .sort-asc', 'sort', 'asc')
 events.bind('click .sort-dsc', 'sort', 'dsc')
 ```
 
+  There is also a shorthand syntax
+  for binding multiple events with one call:
+
+```js
+events.bind({
+  'click .remove': 'remove',
+  'click .close', 'hide'
```
Comment (kamilogorek) [inline README.md]: missing colon

**Comment by everyonesdesign** [comment]
@kamilogorek 
Fixed the readme colon issue (8501c83db31342f4dde7f5e34de7bf13160a05b1), thank you for review

**Comment by HenrikJoreteg** [comment]
@everyonesdesign good question... fixed.

**Comment by everyonesdesign** [comment]
@HenrikJoreteg
So, what do you think about this MR?

**Comment by HenrikJoreteg** [comment]
Hey @everyonesdesign, looks fine to me, i gave you collaborator access on github and npm publishing privileges. Feel free to merge/publish. Please follow semver.

**Comment by everyonesdesign** [comment]
Thank you, I'll merge  it in a week I think.

**Pull request merged**